validation/src/main/scala/hmda/validation/engine/lar/LarEngine.scala

```diff
@@ -1,19 +1,22 @@
 package hmda.validation.engine.lar
 
 import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.model.institution.Institution
```

Comment:
This import is unused, here and in `LarSyntacticalEngine`, `S025`, and `TsSyntacticalEngine`. And a couple of other Engines too.